src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/model/entity/Visit.kt

```diff
@@ -61,6 +62,11 @@ class Visit(
   @Column(nullable = false)
   @Enumerated(EnumType.STRING)
   var visitRestriction: VisitRestriction,
+
+  @Enumerated(EnumType.STRING)
```

Comment:
do we need the 

@Column(nullable = false) annotation? As this seems like it will now be a required field?

Reply:
yes should follow the DB column....